packages/hawtio/src/plugins/shared/workspace.ts

```diff
@@ -13,6 +13,10 @@ export type MBeanCache = { [propertyList: string]: string }
 
 class Workspace {
   private tree: Promise<MBeanTree>
+  private pluginRegisterHandle: number | null = null
```

Comment:
To avoid the overuse of number | null one can put `private pluginRegisterHandle? : number` and just check if it's set or not. Seeing as you are using null as a flag value, you can use undefined as well and use isSet() later

Reply:
Noted and refactored. Thanks.
Cannot use `isSet` from `util/types` due to an error in webpack as described [here](https://bobbyhadz.com/blog/module-not-found-error-cant-resolve-assert#module-not-found-error-cant-resolve-util). However, could not get the workaround config to solve it. Therefore, backed out for the moment